vita3k/shader/include/shader/usse_types.h

```diff
@@ -436,7 +436,7 @@ struct ProgramInput {
 };
 
 enum class ShaderPhase {
-    SampleRate, // Secondary phase
+    SampleRate = 0, // Secondary phase
```

Comment:
```suggestion
    SampleRate, // Secondary phase
```
No need for the = 0, enum fields start by default at 0. Usually one sets a specific value for an enum field when it's not the expected one.

Reply:
It's an habit of mine to explicit at lest the first enum value when some code relies on it but ye that's purely stylistic.